core/src/main/java/com/aem/demo/core/servlets/LikeDislikeServlet.java

```diff
@@ -0,0 +1,53 @@
+package com.aem.demo.core.servlets;
+
+import com.aem.demo.core.services.LikeDislikeService;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.HttpConstants;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import javax.servlet.Servlet;
+import java.io.IOException;
+
+@Slf4j
+@Component(service = Servlet.class,
+        property = {
+                Constants.SERVICE_DESCRIPTION + "= Like and Dislike Servlet",
+                "sling.servlet.methods=" + HttpConstants.METHOD_POST,
+                "sling.servlet.paths=" + "/services/reaction",
+                "sling.servlet.extensions=" + "json"
+        })
+public class LikeDislikeServlet extends SlingAllMethodsServlet {
+    private static final String LIKE = "like";
+    private static final String DISLIKE = "dislike";
+
+    @Reference
+    LikeDislikeService service;
+
+    protected void doPost(final SlingHttpServletRequest request, final SlingHttpServletResponse response) throws IOException {
+        boolean reaction = Boolean.parseBoolean(request.getParameter("reaction"));
```

Comment:
See: `parseBoolean` treats the string "true" as `true` and anything else as false.
![image](https://user-images.githubusercontent.com/48795460/172703394-8f9317be-27ad-43ce-83dc-24fc931e61c7.png)
Therefore it will be a dislike even if "reaction" parameter is missing at all.
This is not a good REST API) Would confuse someone trying to do a request with e.g. Postman. 
Why not using explicit, self-speaking values like `reaction=like` or `reaction=dislike`?